packages/manager/modules/pci/src/projects/project/training/dashboard/install/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,35 @@
+{
+  "pci_projects_project_training_install_section_1_title": "Installation du client",
+  "pci_projects_project_training_install_section_2_title": "Authentification",
+  "pci_projects_project_training_install_section_3_title": "Commandes",
+  "pci_projects_project_training_install_section_1_command_message_1": "Se placer dans le répertoire d'installation voulu et lancer la commande suivante vous permettra de télécharger le client ElementAI sur votre poste de travail et d'installer la configuration OVH :",
+  "pci_projects_project_training_install_section_1_command_message_2": "Aprés l'installation, l'executable ",
+  "pci_projects_project_training_install_section_1_command_message_3": " sera disponible dans le répertoire d'installation",
+  "pci_projects_project_training_install_section_1_command_message_4": "Pour finaliser votre installation ajouter cette variable d'environnement: ",
+  "pci_projects_project_training_install_section_1_command_message_5": "export EAI_PROFILE=GRA",
```

Comment:
i think this shouldn't be a translation string since it's plain code, you don't want this to be translated

Reply:
fixed